platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/FallenBehindManagerImpl.java

```diff
@@ -70,7 +70,7 @@ public FallenBehindManagerImpl(
         }
         this.notifyPlatform = notifyPlatform;
         this.fallenBehindCallback = fallenBehindCallback;
-        this.settings = settings;
+        this.config = config;
```

Comment:
please do not do the null check in one line and the assignment in another line. Since `notifyPlatform`, `fallenBehindCallback`, `config` are stored in local fields do it like this at the beginning of the constructor:

```
 this.notifyPlatform = ArgumentUtils.throwArgNull(notifyPlatform, "notifyPlatform");
...
```